app/Course.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Course extends Model
+{
+    public function professors() {
```

Comment:
como dice @osdagoso, esto va a ser una sola tabla de users con un role. Tengo entendido que el #2 es el role de students. @hecerinc - can you confirm?

Reply:
Can confirm